### DISCUSSION TITLE
`-proxy` support abnormal proxy hostname

### DISCUSSION THREAD

**Discussion by leveryd** [projectdiscovery/katana#256] [Ideas]
in kubernetes situation, i want to pass `katana -proxy http://k8s_servicename`.

`http://k8s_servicename` is not a standard url, because katana will parse url, so it fail.

**Comment by MetzinAround** [comment]
@ehsandeep not sure if you caught this a few months ago, k8s in katana.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
